mqtt-protocol/src/jsMain/kotlin/it/unibo/pulvreakt/mqtt/MqttClient.kt

```diff
@@ -0,0 +1,69 @@
+@file:Suppress("REDUNDANT_NULLABLE")
+@file:JsModule("mqtt")
+@file:JsNonModule
+
+package it.unibo.pulvreakt.mqtt
+
+/**
+ * Represents a MQTT client.
+ */
+external class MqttClient {
+
+    /**
+     * Publishes a message on the given topic.
+     * @param topic The topic.
```

Comment:
In Kotlin, when documenting a function, please use the markdown-like flavor.
https://kotlinlang.org/docs/kotlin-doc.html#inline-markup